templates/taxi/driver_list.html

```diff
@@ -4,10 +4,16 @@
   <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
-      {% for driver in driver_list%}
+      {% for driver in driver_list %}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver == user %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">
```

Comment:
Inconsistent use of quotation marks in the template tag. Consider using single quotes consistently for URL template tags, like `{% url 'taxi:driver-detail' pk=driver.id %}`.